cpp/Makefile

```diff
@@ -1,4 +1,16 @@
 
+CC        = $(CROSS_COMPILE)gcc
+AS        = $(CROSS_COMPILE)as
+CXX       = $(CROSS_COMPILE)g++
+AR        = $(CROSS_COMPILE)ar
+LD        = $(CROSS_COMPILE)ld
+NM        = $(CROSS_COMPILE)nm
+OBJCOPY   = $(CROSS_COMPILE)objcopy
+OBJDUMP   = $(CROSS_COMPILE)objdump
+RANLIB    = $(CROSS_COMPILE)ranlib
+ELF2FLT   = $(CROSS_COMPILE)elf2flt
+STRIPTOOL = $(CROSS_COMPILE)strip
+
```

Comment:
It loks like the only ones really used are `CC`, `AR`, and `RANLIB`. Are the rest of them really necessary? By the way, `RANLIB`gets overwritten at line [18](https://github.com/ev3dev/ev3dev-lang/blob/2bdc5851d0ccdf230f0baba6c0b09ace42e28165/cpp/Makefile#L18) below.

Also, as note to myself: `CC` should be replaced with `CXX`.